src/components/events/EventForm.jsx

```diff
@@ -33,7 +35,12 @@ class EventForm extends React.Component {
 
   handleChange = (e) => {
     const { name, value } = e.target;
-    this.setState({ [name]: value });
+    if (name === 'eventDate') {
+      this.setState({ eventDate: value.substring(8,10) + '/' + value.substring(5,7) + '/' + value.substring(0,4)});
```

Comment:
What's the format that these values are coming in as? This Substring function seems to be assuming formatting of the input.

Reply:
The formatting in the textfield ensures that the format of value is DD/MM/YYYY. I have added a comment above line 39 to inform future developers of the format of value.